### DISCUSSION TITLE
Just Saying

### DISCUSSION THREAD

**Discussion by dasTholo** [asyncgui/asynckivy#62] [General]
I would say a big thank you to you @gottadiveintopython !
I use your program for my gui. With Kivy, KivyMD, MQTT,Redis (asyncClient) and a few Raspberry Pi Stuff. Thanks a lot! I have not yet understood all the possibilities (hobby programmer) but what I do I have to thank to a large extent your work.

Thanks and more of it with pleasure!

**Comment by gottadiveintopython** [comment]
Glad to hear that :heart:
